### PR TITLE
fix(e2e): Make ButtonHero loaded when exchange disabled

### PR DESCRIPTION
**Pull request by AntonioVentilii**
# Note

This is a duplication of #3314 , because I changed my account.

# Motivation

The buttons in the Hero were permanently loading when the Exchange is are not enabled (never initialized). That mostly happens in test environments, such as the one used for E2E test. We can see the artifacts of [workflow E2E test ran on main](https://github.com/dfinity/oisy-wallet/actions/runs/11662656812):

Screenshot 2024-11-04 at 18 43 08

To solve it, we disable the exchanges.

# Changes

- Create env file for E2E tests with disabled exchanges.
- Add the file to playwright configs.

# Tests

I run the [screenshots workflow on this branch](https://github.com/dfinity/oisy-wallet/actions/runs/11684626400) and it worked.

### CODE REVIEW

**Comment by AntonioVentilii** [comment]
@peterpeterparker continuing discussion in [3314](https://github.com/dfinity/oisy-wallet/pull/3314#discussion_r1829302290):

mmmm probably you are right... in the end, that derived store shall say that the exchange are "initialized" if they are not enabled

File: src/frontend/src/lib/components/hero/HeroContent.svelte [hunk 1]
```diff
@@ -45,7 +46,9 @@
 	});
 
 	$: loading.set(
-		isRouteTransactions($page) ? isNullish(pageTokenUi?.balance) : !$exchangeInitialized
+		isRouteTransactions($page)
+			? isNullish(pageTokenUi?.balance)
+			: !EXCHANGE_DISABLED && !$exchangeInitialized
```
[the remark this answers is not in the document]
Comment (AntonioVentilii) [reply src/frontend/src/lib/components/hero/HeroContent.svelte:51]: I created a PR for this, please have a look, we can discuss it directly there, it seems kind of a separate topic (i will remove it from here afterwards)

- PR #3363

**Comment by AntonioVentilii** [comment]
@peterpeterparker ready for another review

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**